install_config/topics/ocp_osp_enabling_octavia.adoc

```diff
@@ -51,7 +51,7 @@ Hat OpenStack Platform release installed.
 ====
 
 The following step pulls the container images from registry.access.redhat.com
```

Comment:
> The Red Hat Registry is represented by two locations:
`registry.access.redhat.com`, which is unauthenticated and deprecated, and
`registry.redhat.io`, which requires authentication. 

registry.access.redhat.com should probably be `registry.redhat.io`, in backticks.

Reply:
Good catch! I just updated something like this in the 3.11 release notes last week.